### DISCUSSION TITLE
TrueNAS integration?

### DISCUSSION THREAD

**Discussion by DatGeekUKnow** [DouwJacobs/plex-shuffler#112] [Q&A]
Hey there 👋

I will start by saying I'm not very familiar with the docker environment in general.

I use the Plex App on my TrueNAS SCALE machine. 
Is there a handy way I can add this to the TrueNAS Plex app?

I think TrueNAS is using docker under the hood, but again I'm not very familiar because it's encapsulated by the TrueNAS UI

**Comment by DouwJacobs** [comment]
Hi, 😊

I don't have any experience on TrueNAS. I've found this guide:
https://www.truenas.com/docs/scale/22.12/scaletutorials/apps/docker/

Maybe check if you can deploy the docker image using this guide?
